### PR TITLE
(DOCSP-24560): Sync data from multiple processes

### PR DESCRIPTION
**Pull request by mongodben**
## Pull Request Info

Use a synced realm across multiple processes

### Jira

- https://jira.mongodb.org/browse/DOCSP-24560

### Staged Changes (Requires MongoDB Corp SSO)

- [Sync Multiple Processes](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-24560/sdk/flutter/sync/sync-multiple-processes/)

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/sdk/flutter/sync/sync-multiple-processes.txt [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+- A multi-window desktop application where each window writes to the same synced realm.
+- A server application that writes to a synced realm from multiple processes.
+
+On a high level, syncing data to a realm from multiple processes consists
+of the following:
+
+#. Creating a single **main process** that opens a Realm using a standard flexible
+   sync configuration. The main process handles synchronization.
+#. Creating one or more **secondary processes** that open the same realm using a
+   disconnected sync configuration. The secondary processes can read
```
Comment (mongomoe) [inline source/sdk/flutter/sync/sync-multiple-processes.txt:22]: What is a disconnected sync configuration?

File: source/sdk/flutter/sync/sync-multiple-processes.txt [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+process include:
+
+- A multi-window desktop application where each window writes to the same synced realm.
+- A server application that writes to a synced realm from multiple processes.
+
+On a high level, syncing data to a realm from multiple processes consists
+of the following:
+
+#. Creating a single **main process** that opens a Realm using a standard flexible
+   sync configuration. The main process handles synchronization.
```
Comment (mongomoe) [inline source/sdk/flutter/sync/sync-multiple-processes.txt:20]: So the secondary processes write to a realm and notify the main process & data can only be synced from the single main process?

File: source/sdk/flutter/sync/sync-multiple-processes.txt [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+=================================
+Sync Data from Multiple Processes
+=================================
+
+You can use a single realm database to synchronize data from multiple processes
+using Atlas Device Sync.
+
+Examples of scenarios where you might want to open a synced realm in a multiple
+process include:
```
Comment (mongomoe) [inline source/sdk/flutter/sync/sync-multiple-processes.txt:11]: Required fix:
"Examples of scenarios where you might want to open a synced realm in a multiple process include:" 

The fragment "in a multiple process" isn't correct. Did you mean: "in multiple processes"?

**Review by mongomoe — COMMENTED** [review]
Great work so far, 1 needed fix + a couple questions.

File: source/sdk/flutter/sync/sync-multiple-processes.txt (hunk 2 above)
Comment (mongodben) [reply source/sdk/flutter/sync/sync-multiple-processes.txt:20]: yes

File: source/sdk/flutter/sync/sync-multiple-processes.txt (hunk 3 above)
Comment (mongodben) [reply source/sdk/flutter/sync/sync-multiple-processes.txt:11]: yes

File: source/sdk/flutter/sync/sync-multiple-processes.txt (hunk 1 above)
Comment (mongodben) [reply source/sdk/flutter/sync/sync-multiple-processes.txt:22]: that's the technical term the dart sdk team is using -https://pub.dev/documentation/realm/latest/realm/Configuration/disconnectedSync.html

i will clarify the text to be clearer on what this means

File: source/sdk/flutter/sync/sync-multiple-processes.txt [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+- A multi-window desktop application where each window writes to the same synced realm.
+- A server application that writes to a synced realm from multiple processes.
+
+On a high level, syncing data to a realm from multiple processes consists
+of the following:
+
+#. Creating a single **main process** that opens a Realm using a standard flexible
+   sync configuration. The main process handles synchronization.
+#. Creating one or more **secondary processes** that open the same realm using a
+   disconnected sync configuration. Using a disconnected sync configuration,
```
Comment (mongomoe) [inline source/sdk/flutter/sync/sync-multiple-processes.txt:22]:
```suggestion
   disconnected sync configuration. If you use a disconnected sync configuration,
```

The fragment "Using a disconnected sync configuration" reads kinda weirdly, and makes you wonder 'who is using'

**Review by mongomoe — APPROVED** [review]
LGTM

**Pull request merged**